contrib/base64-cmake/CMakeLists.txt

```diff
@@ -11,50 +11,74 @@ endif()
 
 SET(LIBRARY_DIR "${ClickHouse_SOURCE_DIR}/contrib/base64")
 
-add_library(_base64_scalar OBJECT "${LIBRARY_DIR}/turbob64c.c" "${LIBRARY_DIR}/turbob64d.c")
-add_library(_base64_ssse3 OBJECT "${LIBRARY_DIR}/turbob64sse.c") # This file also contains code for ARM NEON
-
-if (ARCH_AMD64)
-    add_library(_base64_avx OBJECT "${LIBRARY_DIR}/turbob64sse.c") # This is not a mistake. One file is compiled twice.
-    add_library(_base64_avx2 OBJECT "${LIBRARY_DIR}/turbob64avx2.c")
+set(HAVE_NEON32 0)
```

Comment:
Looking at l. 1-5, I think we don't need the restriction to x86/ARM/PPC/s390 any more. Not that there are major other platforms (maybe RISC-V) but base64 should fall back to a generic algorithms on these architectures.